apps/src/codebridge/FileBrowser/hooks/useHandleFileUpload.ts

```diff
@@ -20,6 +22,10 @@ type handleFileUploadArgs = {
 
 export const useHandleFileUpload = (files: Record<string, ProjectFile>) => {
   const appName = useAppSelector(state => state.lab.levelProperties?.appName);
+  const isStartMode = getAppOptionsEditBlocks() === START_SOURCES;
+  const validationFile = useAppSelector(
+    state => state.lab.levelProperties?.validationFile
+  );
```

Comment:
just double checking - even if there is more than one test, there will always only be one validation file?